libs/wingsdk/src/core/tree.ts

```diff
@@ -33,6 +34,34 @@ export interface ConstructTreeNode {
    * Information on the construct class that led to this node, if available.
    */
   readonly constructInfo?: ConstructInfo;
+
+  /**
+   * Information on how to display this node in the UI.
+   */
+  readonly display?: DisplayInfo;
```

Comment:
Instead of adding a new field here, could we add this information as another attribute (like the existing connections and stateful attributes)? This can be done by updating the `_inspect` method:

https://github.com/winglang/wing/blob/aa8e4510cd508ed38213d0afda65d6f4cc860cd1/libs/wingsdk/src/core/resource.ts#L196-L208

(sorry in advance, this needs to be better documented :))

Reply:
@Chriscbr I am curious, what's the value you see in using these attributes instead of simply extending the base schema?

Reply:
@eladb I guess there isn't much of a difference - my thought process back in the day was "it looks like tree.json in the CDK puts most optional metadata in this `attributes` field, so if we want to play well, we can put stuff there too." There's an also an argument that by putting it in this `attributes` field, it signals more clearly that the attribute isn't required / isn't guaranteed to exist. I'm ok with the current place if you feel it makes more sense though 👍

Reply:
I feel there's value in lifting up everything into the schema as first class citizens and getting rid of "attributes" as a 2nd class protocol.